common/djangoapps/student/views/management.py

```diff
@@ -1034,18 +1043,42 @@ def do_email_change_request(user, new_email, activation_key=None):
         log.error(u'Unable to send email activation link to user from "%s"', from_address, exc_info=True)
         raise ValueError(_('Unable to send email activation link. Please try again later.'))
 
-    # When the email address change is complete, a "edx.user.settings.changed" event will be emitted.
-    # But because changing the email address is multi-step, we also emit an event here so that we can
-    # track where the request was initiated.
-    tracker.emit(
-        SETTING_CHANGE_INITIATED,
-        {
-            "setting": "email",
-            "old": message_context['old_email'],
-            "new": message_context['new_email'],
-            "user_id": user.id,
-        }
-    )
+    if not secondary_email_change_request:
```

Comment:
Do we need to track event for secondary email change as well?

Reply:
Not required you.